From 41eba90bb6b00c35547604378d29d67ba37e84ad Sun, 20 Jun 2021 16:37:20 +0200
From: Enrico Horn <farmboy0+git@googlemail.com>
Date: Sat, 12 Apr 2014 17:00:02 +0200
Subject: [PATCH] disable all CLI commands with ENABLE_CLI=OFF

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 597503d..0bc2d7a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -527,8 +527,6 @@
 add_subdirectory (dispatch)
 add_subdirectory (helpers)
 add_subdirectory (wrappers)
-add_subdirectory (retrace)
-add_subdirectory (frametrim)
 
 
 ##############################################################################
@@ -540,6 +538,8 @@
     endif ()
     add_subdirectory (cli)
     add_subdirectory (scripts)
+    add_subdirectory (retrace)
+    add_subdirectory (frametrim)
 endif ()
 
 
