diff --git a/CMakeLists.txt b/CMakeLists.txt
index 200cc24..0148ae0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -501,7 +501,6 @@
 add_subdirectory (helpers)
 add_subdirectory (wrappers)
 add_subdirectory (image)
-add_subdirectory (retrace)
 
 
 ##############################################################################
@@ -513,6 +512,7 @@
     endif ()
     add_subdirectory (cli)
     add_subdirectory (scripts)
+    add_subdirectory (retrace)
 endif ()
 
 
