diff --git a/CMakeLists.txt b/CMakeLists.txt
index de0e5d1..13b1270 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -537,7 +537,6 @@
 add_subdirectory (dispatch)
 add_subdirectory (helpers)
 add_subdirectory (wrappers)
-add_subdirectory (retrace)
 
 
 ##############################################################################
@@ -549,6 +548,7 @@
     endif ()
     add_subdirectory (cli)
     add_subdirectory (scripts)
+    add_subdirectory (retrace)
 endif ()
 
 
