diff --git a/CMakeLists.txt b/CMakeLists.txt
index bfb69ad..9882a0b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -486,7 +486,6 @@
 add_subdirectory (helpers)
 add_subdirectory (wrappers)
 add_subdirectory (image)
-add_subdirectory (retrace)
 
 
 ##############################################################################
@@ -497,32 +496,35 @@
         add_subdirectory (inject)
     endif ()
     add_subdirectory (cli)
+    add_subdirectory (retrace)
 endif ()
 
 ##############################################################################
 # Scripts (to support the CLI)
 
-install (
-    PROGRAMS
-        scripts/highlight.py
-        scripts/jsondiff.py
-        scripts/profileshader.py
-        scripts/retracediff.py
-        scripts/snapdiff.py
-        scripts/tracecheck.py
-        scripts/tracediff.py
-        scripts/unpickle.py
-    DESTINATION ${SCRIPTS_INSTALL_DIR}
-)
-if (WIN32)
+if (ENABLE_CLI)
     install (
-        PROGRAMS scripts/convert.py
+        PROGRAMS
+            scripts/highlight.py
+            scripts/jsondiff.py
+            scripts/profileshader.py
+            scripts/retracediff.py
+            scripts/snapdiff.py
+            scripts/tracecheck.py
+            scripts/tracediff.py
+            scripts/unpickle.py
         DESTINATION ${SCRIPTS_INSTALL_DIR}
     )
-    install (
-        FILES scripts/apitrace.PIXExp
-        DESTINATION ${SCRIPTS_INSTALL_DIR}
-    )
+    if (WIN32)
+        install (
+            PROGRAMS scripts/convert.py
+            DESTINATION ${SCRIPTS_INSTALL_DIR}
+        )
+        install (
+            FILES scripts/apitrace.PIXExp
+            DESTINATION ${SCRIPTS_INSTALL_DIR}
+        )
+    endif ()
 endif ()
 
 ##############################################################################
