diff --git a/meson.build b/meson.build
index 41808fc..78a36e0 100644
--- a/meson.build
+++ b/meson.build
@@ -42,8 +42,6 @@ cap_dep = dependency('libcap')
 sdl_dep = dependency('SDL2')
 pipewire_dep = dependency('libpipewire-0.3', required: get_option('pipewire'))
 
-stb_dep = dependency('stb')
-
 wlroots_dep = dependency(
   'wlroots',
   version: ['>= 0.15.0', '< 0.16.0'],
@@ -115,7 +113,8 @@ executable(
     dep_x11, dep_xdamage, dep_xcomposite, dep_xrender, dep_xext, dep_xfixes,
     dep_xxf86vm, dep_xres, drm_dep, wayland_server, wayland_protos,
     xkbcommon, thread_dep, sdl_dep, wlroots_dep,
-    vulkan_dep, liftoff_dep, dep_xtst, cap_dep, pipewire_dep, stb_dep,
+    vulkan_dep, liftoff_dep, dep_xtst, cap_dep, pipewire_dep,
   ],
   install: true,
+  include_directories : include_directories('/usr/include/stb'),
 )
