diff --git a/meson.build b/meson.build
index 41808fc..78a36e0 100644
--- a/meson.build
+++ b/meson.build
@@ -49,9 +49,6 @@
 librt_dep = cppc.find_library('rt', required : get_option('pipewire'))
 hwdata_dep = dependency('hwdata')
 
-stb_dep = dependency('stb')
-vkroots_dep = dependency('vkroots')
-
 wlroots_dep = dependency(
   'wlroots',
   version: ['>= 0.16.0', '< 0.17.0'],
@@ -142,8 +139,6 @@
 
 subdir('protocol')
 
-subdir('layer')
-
 executable(
   'gamescope',
   src,
@@ -152,7 +147,7 @@
     dep_xxf86vm, dep_xres, drm_dep, wayland_server,
     xkbcommon, thread_dep, sdl_dep, wlroots_dep,
     vulkan_dep, liftoff_dep, dep_xtst, cap_dep, pipewire_dep, librt_dep,
-    stb_dep,
   ],
   install: true,
+  include_directories : include_directories('/usr/include/stb'),
 )
